From 15fc71cc82f2dc93321b60d557efdc968f5d9433 Mon Sep 17 00:00:00 2001
From: Breno Leitao <breno.leitao@gmail.com>
Date: Tue, 9 May 2017 16:54:49 +0000
Subject: [PATCH] Define PATH_MAX if not defined

On systems without glibc, you might lack definition of PATH_MAX.
Defining it as already defined it for interface_http.c as defined in
civetweb.c.
---
 src/modules/interface/http/interface_http.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/modules/interface/http/interface_http.c b/src/modules/interface/http/interface_http.c
index 5c71e8e..4686c89 100644
--- a/src/modules/interface/http/interface_http.c
+++ b/src/modules/interface/http/interface_http.c
@@ -53,6 +53,10 @@
 #include <json.h>  
 #endif
 
+#ifndef PATH_MAX
+#define PATH_MAX 4096
+#endif
+
 #include <captagent/globals.h>
 #include <captagent/api.h>
 #include <captagent/structure.h>
-- 
2.12.2

